nixos/modules/virtualisation/amazon-options.nix

```diff
@@ -18,4 +59,17 @@
       };
     };
   };
+
+  config = lib.mkIf config.ec2.zfs.enable {
+    networking.hostId = lib.mkDefault "00000000";
```

Comment:
I seem to recall hostId being an important meaningful data for ZFS. How can setting a default hostId bite end-users?

Reply:
I believe hostId is just to differentiate between pools that were previously imported on another machine (and not properly exported). Not entirely certain, though.

Reply:
ZFS refuses to import pools unless the host's host ID matches the host ID  stored in the pool. However, we force import the pools by default anyway.